content/docs/pipelines/upgrade.md

```diff
@@ -0,0 +1,218 @@
+[Persistent Disk](https://cloud.google.com/persistent-disk/) (PD)
+and mounts it as a PV. 
+* If you are not deploying to GCP, you can specify your own prefered PV.
+
+## Deploying Kubeflow
+
+This section describes how to deploy Kubeflow in a way that ensures you can use
+the Kubeflow Pipelines upgrade/installation capability.
+
+### Deploying Kubeflow on GCP 
+
+Follow the guide to [getting started with Kubeflow on
+GKE](/docs/started/getting-started-gke/). You don't need to do anything extra. 
+
+When the deployment has finished, you can see two entries in the GCP 
+Deployment Manager, one for deploying the cluster and one for
+deploying the storage:
+
+<img src="/docs/images/pipelines-deployment-storage1.png" 
+  alt="Deployment Manager showing the storage deployment entry"
+  class="mt-3 mb-3 border border-info rounded">
+
+The entry suffixed with `-storage` creates one PD for the metadata store and one
+for the artifact store:
+
+<img src="/docs/images/pipelines-deployment-storage2.png" 
+  alt="Deployment Manager showing details of the storage deployment entry"
+  class="mt-3 mb-3 border border-info rounded">
+
+### Deploying Kubeflow in other environments (non-GCP) 
+
+The steps below assume that you already have a Kubernetes cluster set up.
+
+* If you don't need custom storage and are happy with the default PVs that
+  Kubeflow provides, you can follow the Kubeflow
+  [quick start](/docs/started/getting-started/#kubeflow-quick-start)
+  without doing anything extra. The deployment script uses the Kubernetes 
+  default
+  [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/#the-storageclass-resource)
+  to provision the PVs for you. 
+
+* If you want to specify a custom PV:
+
+  1. Create two PVs in your Kubernetes cluster with your preferred storage type. 
+     See the
+     [Kubernetes guide to PVs](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes).  
+
+  1. Follow the Kubeflow
+     [quick start](/docs/started/getting-started/#kubeflow-quick-start),
+     but note the following change to the standard procedure:
+
+        **Before** running the `apply` command:
+
+        ```
+        ${KUBEFLOW_SRC}/scripts/kfctl.sh apply k8s
+        ```
+
+        You must run the following commands to specify your PVs:
+
+        ```
+        cd ks_app
+        ks param set pipeline mysqlPvName [YOUR-PRE-CREATED-MYSQL-PV-NAME]
+        ks param set pipeline minioPvName [YOUR-PRE-CREATED-MYSQL-MINIO-NAME]
+        ```
+
+  1. Then run the `apply` command as usual:
+
+        ```
+        ${KUBEFLOW_SRC}/scripts/kfctl.sh apply k8s
+        ``` 
+
+## Upgrading your Kubeflow Pipelines deployment
+
+To upgrade your Kubeflow Pipelines deployment, run the following script in the 
+Kubeflow application directory. That is, in the same directory where you 
+performed the original deployment, represented in the deployment guide as
+`${KFAPP}`:
+
+```
+cd ${KUBEFLOW_SRC}/${KFAPP}
```

Comment:
I think ${KFAPP} is not necessary under ${KUBEFLOW_SRC}. 
here  `cd ${KFAPP}` should be good enough

Reply:
Done.